internal/model/rwfolder.go

```diff
@@ -1082,3 +1093,10 @@ func removeDevice(devices []protocol.DeviceID, device protocol.DeviceID) []proto
 	}
 	return devices
 }
+
+func moveForConflict(name string) error {
+	ext := filepath.Ext(name)
+	withoutExt := name[:len(name)-len(ext)]
+	newName := fmt.Sprintf("%s (Syncthing conflicted %s)%s", withoutExt, time.Now().Format("2006-01-02 150405"), ext)
```

Comment:
This is a very long file name, plus full of spaces which is not terminal friendly.